### PR TITLE
Create private-repositories.md

### PR DESCRIPTION
**Pull request by rebeccaradding**
Related to https://github.com/foundersandcoders/international/issues/135, foundersandcoders/london-programme#199

- Creating this file in `master-reference` so it will be publicly available, will link to it from `international` once merged
- Does the file name reflect the contents?
- No links yet to associate membership, once a process exists I will revisit this file and add links

### CODE REVIEW

File: private-repositories.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+## Private repositories
+
+Founders & Coders also uses GitHub to coordinate business specific to the running of each campus and the running of our global charity (currently being incorporated), Founders & Coders International. Frequently, coordination of this nature involves raising issues where it is necessary to share personal information about students and clients. Given the regions in which we operate international programmes (Israel and Gaza), we also find ourselves in a position where participant privacy is directly related to participant safety. 
+
+For these reasons, the following Founders & Coders GitHub repositories are **private** (as of September 2017):
+
+_London:_
+- `london-programme`
+- `london-employers`
+- `london-money`
```
Comment (shiryz) [inline private-repositories.md]: London money is not a private repo! (shouldn't it be? :fearful:)

Comment (iteles) [reply private-repositories.md]: I have _always_ advocated that it should remain public for a CIC or charity.

File: private-repositories.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- (almost) all projects hosted by the Founders & Coders organisation (such as [`FAC-share`](https://github.com/foundersandcoders/FAC-Share))
+
+## Private repositories
+
+Founders & Coders also uses GitHub to coordinate business specific to the running of each campus and the running of our global charity (currently being incorporated), Founders & Coders International. Frequently, coordination of this nature involves raising issues where it is necessary to share personal information about students and clients. Given the regions in which we operate international programmes (Israel and Gaza), we also find ourselves in a position where participant privacy is directly related to participant safety. 
+
+For these reasons, the following Founders & Coders GitHub repositories are **private** (as of September 2017):
+
+_London:_
+- `london-programme`
```
Comment (iteles) [inline private-repositories.md]: I still don't understand why all of the `-programme` repos are private. Do we intend to _keep_ them private? We're not actually sharing student information in these are we?

File: private-repositories.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+_Nazareth:_
+- `nazareth-programme`
+- `nazareth-employers`
+- `nazareth-money`
+
+_Gaza:_
+- `gaza-programme`
+- `gaza-employment`
+- `gaza-membership`
```
Comment (iteles) [inline private-repositories.md]: Why is gaza membership private?

File: private-repositories.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+_International:_
+- `international`
```
Comment (iteles) [inline private-repositories.md]: Are we planning on open-sourcing this once we have our membership sorted? I didn't even realise it had become private 😕

File: private-repositories.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+- `selection-committee`
+
+_Other:_
+- `graduate-mvps`
+- `testimonial`
```
Comment (iteles) [inline private-repositories.md]: If there was ever a repo that should be ***public***... 😳 
There's a typo here, should be `testimonials` (with an 's')

File: private-repositories.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+- `freelancing`
+- `freelance-networking-and-resources`
+- `old-task-groups`
+
+
+_Client projects:_
```
Comment (iteles) [inline private-repositories.md]: I don't think we need to enumerate this, I would just say something along the lines of 'A small number of client projects who we have not yet been able to convince of the benefits of open source!'

File: private-repositories.md (hunk 2 above)
Comment (rebeccaradding) [reply private-repositories.md]: @iteles legal issues related to travel and visas are frequently discussed in `nazareth` and `gaza` programme. In Israel, we have had students whose phones have been thoroughly searched at border control. In Gaza, any mention of a relationship to an organisation that operates in Israel could raise levels of suspicion and cause problems for course participants, so we were asked to keep all Gaza-related repos private until a better solution could be worked out.

File: private-repositories.md (hunk 3 above)
Comment (rebeccaradding) [reply private-repositories.md]: @iteles see above explanation.

File: private-repositories.md (hunk 4 above)
Comment (rebeccaradding) [reply private-repositories.md]: @iteles at the moment it's private due to the fact that it discusses Gaza and Nazareth in the same place.

Longer term, I'm agnostic regarding the status of the `international` repo, but before making it public, I'd like to be crystal clear on which, if any documents should be private and want to make sure we have clear documentation explaining where to find those documents if they need to be hosted elsewhere.

File: private-repositories.md (hunk 5 above)
Comment (rebeccaradding) [reply private-repositories.md]: @iteles completely agree - but when I compiled this list, I saw it was private, so I added it to the list here. I can certainly change it, as I don't see any reason why it should be private.

File: private-repositories.md (hunk 6 above)
Comment (rebeccaradding) [reply private-repositories.md]: @iteles I'll change the wording.

**Comment by rebeccaradding** [comment]
@iteles thank you so much for your feedback. I've attempted to answer several of your questions, and have fixed the blocking typo.

**Pull request merged**